array.c

```diff
@@ -2101,6 +2101,40 @@ rb_ary_each(VALUE ary)
     return ary;
 }
 
+/*
+ *  call-seq:
+ *     ary.pluck(key)    -> ary
+ *
+ *  Returns an array of values from an input array of hashes
+ *
+ *     a = [ {name: "Lewis"}, {name: "Steffan"} ]
```

Comment:
Cheeky way to get your name in the history books!

Reply:
Shameless self promotion 😉

Reply:
One would say... A shameless pluck. 😎